app/services/AlfrescoStore.java

```diff
@@ -238,7 +228,7 @@ private WSRequest makeRequest(String operation, String onBehalfOfUser) {
 
             Logger.error("Upload error", ex);
 
-            return CompletableFuture.supplyAsync(ImmutableMap::of);
+            throw new RuntimeException(ex);
```

Comment:
Throw exception and allow error page to handle this rather than pretend all went well when it actually failed